buddyboss/buddyboss-platform
# #2768 - fix - Removed the commented line form the flv.js file

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2768.

### How to test the changes in this Pull Request:

1. Go to 'News Feed'
2. Chek on inspect
3. See the error "DevTools failed to load source map: Could not load content for https://dev-riddhi.flywheelsites.com/wp-content/plugins/buddyboss-platform/bp-core/js/vendor/flv.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE"

### Proof Screenshots or Video

![Screenshot 2021-08-11 at 5 15 44 PM](https://user-images.githubusercontent.com/82605214/129028791-c3449f0f-7d19-4ceb-ab4f-e23847cb13d3.png)

![Screenshot 2021-08-11 at 5 19 30 PM](https://user-images.githubusercontent.com/82605214/129028802-72472b16-82d8-411a-9e5c-a0a7b477801b.png)


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
